releasenotes/template.yaml

```diff
@@ -0,0 +1,36 @@
+# This YAML file describes the format for specifying a release notes entry for Istio.
+# This should be filled in for all user facing changes.
+
+# kind describes the type of change that this represents. 
+# Valid Values are:
+# - bug-fix -- Used to specify that this change represents a bug fix. 
+# - security-fix -- Used to specify that this change represents a security fix. 
+# - feature -- Used to specify a new feature that has been added. 
+# - test -- Used to describe additional testing added. This file is optional for
+# tests, but included for completeness.
```

Comment:
Can we indent `tests` two spaces since this line is a continuation of the prior line?

Reply:
Done. Thanks.